test/block-metadata.test.ts

```diff
@@ -0,0 +1,110 @@
+import { mocked } from 'ts-jest/utils'
+import { getContractKit } from '../src/util/utils'
+import { ContractKit } from '@celo/contractkit'
+import { database, initDatabase } from '../src/database/db'
+import { handleBlockMetadata } from '../src/indexer/block-metadata'
+
+jest.mock('../src/util/utils')
+jest.mock('../src/config', () => ({
+  BLOCK_METADATA_DEFAULT_MIN_BLOCK_NUMBER: 100,
+}))
+
+describe('block metadata', () => {
```

Comment:
nit: add a test to ensure that an error on an "intermediate block" prevents subsequent blocks from being indexed? seems good to test since otherwise error'd blocks would never be indexed.

Reply:
added a test